### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mykelalvis**
Currently skips javadoc gen for scala code and upgrades versions on rather a large number of dependencies.

### CODE REVIEW

File: cotiviti-nexgen-parent/pom.xml [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
-    activity or policy violation, Cotiviti personnel may provide the evidence of
-    such monitoring to law enforcement or other officials, and the user may be
-    subject to disciplinary action by Cotiviti, up to and including termination
-    of employment.
-
-    Use of this software code and any applications and information therein
-    constitutes acknowledgement of and consent to this notice
--->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<!-- Copyright © 2015 Cotiviti Labs (nexgen.admin@cotiviti.io) The software
```
Comment (isomarcte) [inline cotiviti-nexgen-parent/pom.xml]: This is going to cause a duplicate license, right?

File: pom.xml [hunk 2]
```diff
@@ -65,9 +65,9 @@
 	<description>Parent POM for Cotiviti projects</description>
 	<properties>
 		<surefire.failIfNoTests>true</surefire.failIfNoTests>
-		<aws.sdk.version>1.11.26</aws.sdk.version>
+		<aws.sdk.version>1.11.29</aws.sdk.version>
 		<functional.java.version>4.6</functional.java.version>
-		<cotiviti.core.support.version>2.0.4</cotiviti.core.support.version>
+		<cotiviti.core.support.version>2.0.5-SNAPSHOT</cotiviti.core.support.version>
```
Comment (isomarcte) [inline pom.xml]: SNAPSHOT in parent?

Comment (mykelalvis) [inline pom.xml]: Yes.  At the moment, that is the correct value.
This isn't releasable yet.

File: cotiviti-nexgen-parent/pom.xml (hunk 1 above)
Comment (mykelalvis) [inline cotiviti-nexgen-parent/pom.xml]: Shouldn't.

File: pom.xml (hunk 2 above)
Comment (isomarcte) [inline pom.xml]: Can you just set this locally in the projects? I feel very uncomfortable with SNAPSHOT values in parent.

Comment (mykelalvis) [inline pom.xml]: 2.0.4 is bad and wrong.  This is simply a prelude to the 2.0.5 releases.

**Pull request merged**